pkgs/development/tools/buf-language-server/default.nix

```
@@ -0,0 +1,27 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "buf-language-server";
+  version = "unstable-2022-09-01";
```

Sorry, didn't notice this at first.

```suggestion
  version = "unstable-2022-08-19";
```


> https://github.com/NixOS/nixpkgs/blob/master/doc/contributing/coding-conventions.chapter.md#package-naming-sec-package-naming
>
> If a package is not a release but a commit from a repository, then the `version` attribute _must_ be the date of that (fetched) commit. The date _must_ be in `"unstable-YYYY-MM-DD"` format.

---

ahh, thanks for this link.